.github/workflows/CI.yml

```diff
@@ -17,14 +18,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version:
-          - '1.7'
-          - '1.8'
-          - '~1.9.0-0'
-        os:
-          - ubuntu-latest
-        arch:
-          - x64
+        version: ['1.6', '1']
+        os: [ubuntu-latest]
+        arch: [x64]
+        allow_failure: [false]
+        include:
+          - version: 'nightly'
+            os: ubuntu-latest
+            arch: x64
+            allow_failure: true
```

Comment:
Added so that failures on nightly (currently due to Zygote precompilation) do not impact the "CI passing" badge